shared/src/commonMain/kotlin/com/bumble/livemosaic/model/Entries.kt

```diff
@@ -75,6 +75,11 @@ val entries = listOf(
         githubUserName = "juanmazake",
         message = "You never lose. You either win or you learn."
     ),
+    Entry.Text(
+        mosaic = MOSAIC2,
```

Comment:
Hello :) Could you please change this to MOSAIC1

Reply:
Hello @mike-n-jordan!
just fixed it)